### PR TITLE
feat: added basic parameter inputs

### PR DESCRIPTION
**Pull request by Jaryt**

### CODE REVIEW

**Review by jliby — APPROVED** [review]
lgtm

**Review by EricRibeiro — CHANGES_REQUESTED** [review]
Leftover?

File: src/components/containers/inspector/JobInspector.tsx [hunk 1]
```diff
@@ -14,7 +14,10 @@ const JobInspector = (
 ) => {
   const navigateTo = useStoreActions((actions) => actions.navigateTo);
   const executor = props.values.executor;
-  const executorName = typeof executor === 'string' ? executor : executor?.name;
+  const isFlat = typeof executor === 'string';
+  const executorName = isFlat ? executor : executor?.name;
+
+  console.log(executorName);
```
Comment (EricRibeiro) [inline src/components/containers/inspector/JobInspector.tsx:20]: Leftover?

Comment (Jaryt) [reply src/components/containers/inspector/JobInspector.tsx:20]: always 🤦

**Pull request merged**